test/test_game.py

```diff
@@ -73,3 +73,21 @@ def test_if_current_player_is_X_then_O_wins(self):
         total_marks_on_board = 6
         winner = game.get_winning_mark(total_marks_on_board)
         self.assertEqual("O", winner)
+
+    @patch("builtins.input", side_effect=["3"])
+    def test_set_player_one_symbol_returns_symbol_with_input_3(self, mock_input):
+        game = Game()
+        result = game.set_player_one_symbol()
+        self.assertEqual("😎", result)
```

Comment:
This is a slightly better setup in the test, because the assertion would no longer be a "magic number"(magic symbol?)
```suggestion
    @patch("builtins.input", side_effect=["3"])
    def test_set_player_one_symbol_returns_symbol_with_input_3(self, mock_input):
        game = Game()
        expected_symbol = SymbolOptions().get_symbol("3")
        result = game.set_player_one_symbol()
        self.assertEqual(expected_symbol, result)
```